locale/ko/docs/guides/blocking-vs-non-blocking.md

```diff
@@ -141,7 +141,7 @@ moreWork(); // will run before console.log
 const fs = require('fs');
 const data = fs.readFileSync('/file.md'); // 파일을 읽을 때까지 여기서 블로킹됩니다.
 console.log(data);
-// moreWork();는 console.log 이후 실행될 것입니다.
+moreWork(); // moreWork();는 console.log 이후 실행될 것입니다.
```

Comment:
```suggestion
moreWork(); // 는 console.log 이후 실행될 것입니다.
```

Reply:
I would prefer this:
```suggestion
moreWork(); // 이는 console.log 이후 실행될 것입니다.
```
How do you think? @nodejs/nodejs-ko